src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -80,9 +91,7 @@ export default function AccountRecord({
               <TransferDetails account={record.details.account} />
             )}
             {showStatusNote(record, t)}
-            {(record?.details?.donorCertificate ||
-              record?.details?.taxDeductibleReceipt ||
-              record?.details?.giftCertificate) && (
+            {showCertificate && record?.purpose !== 'conservation' && (
```

Comment:
I don't understand why you do not put `record?.purpose !== 'conservation'` as a part of the `showCertificate` state?  `showCertificate` is not used for anything else, is it?

Reply:
previously ,I tried the same , but I got the codefactor error (complex method). to fix that error , I wrote this logic

Reply:
@sunilsabatp Adding this check `record?.purpose !== 'conservation'` should not affect the complexity much. 

If codefactor is showing a warning in this case, you can ignore it. Codefactor warnings are useful as a guide, but exercise your own judgement while following them.